src/main/java/jeresources/entry/VillagerEntry.java

```diff
@@ -36,7 +36,8 @@ public void addITradeLists(Int2ObjectMap<VillagerTrades.ITrade[]> tradesLists) {
     }
 
     public TradeList getVillagerTrades(int level) {
-        return tradeList.get(level);
+        if (tradeList.size() > 0) return tradeList.get(level);
```

Comment:
This can still go out of bounds, per example, tradeList.size() = 3 and requested level is 5.

We'd need a check like `tradeList.size() >= level`

Reply:
Yes, it's better (but >, not >=. Indexing from zero), but out of bounds occurs only when this code is executed: https://github.com/way2muchnoise/JustEnoughResources/blob/d4781c3a57e794e3d20b05750f7b507ecc0b1cfa/src/main/java/jeresources/registry/VillagerRegistry.java#L25